src/webgl/program.js

```diff
@@ -86,19 +86,19 @@ export default class Program extends Resource {
     transformFeedback = null,
     uniforms = {},
     samplers = {},
-    parameters = {}
+    parameters = null
   }) {
     vertexArray = vertexArray || VertexArray.getDefaultArray(this.gl);
     vertexArray.bind(() => {
 
       this.gl.useProgram(this.handle);
 
-      if (transformFeedback) {
-        if (parameters[GL.RASTERIZER_DISCARD]) {
-          // bypass fragment shader
-          this.gl.enable(GL.RASTERIZER_DISCARD);
-        }
+      if (parameters) {
+        log.error('Program.draw({prameters}) not needed to disable RASTERIZATION, \
```

Comment:
I am not sure about this. 

It is not too hard to imagine cases when one wants to both draw and get transform feedback. Forcing it always off seems a bit too strict, we are taking something away from the user.

Shutting off the rasterizer by default could make sense though, if it is clearly documented. Just not forcing it off.

Reply:
Agree, I will make this default behavior with a flag that disables it.